docs/src/DeveloperDocumentation/new_developers.md

```diff
@@ -295,3 +295,133 @@ git fetch oscar-system
 git rebase oscar-system/master
 ```
 Adding the remote only has to be executed once.
+
+### Pitfalls: Mutable objects in Oscar code
+
+The objects in Oscar that can be changed (mutable objects) can be linked
```

Comment:
But this isn't Oscar specific, isn't it? This is affects all of Julia (or for that matter, Python, Ruby, Java, GAP, ...)

Reply:
I changed Oscar->Julia